recipes/spleeter/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "spleeter" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: fea6b1748e285cdef564d439f14500ed4a6c5ce8721f440e1dd7ae656749aaae
+  - url: https://pypi.io/packages/source/n/norbert/norbert-0.2.1.tar.gz
```

Comment:
`norbert`, `musdb` and `museval` should be added as new packages in conda-forge before this is merged.

Reply:
Alright, I will create package for them and let you know as long as it is done :)

Reply:
I already created following PR for `norbert` and `musdb` Python package :

- https://github.com/conda-forge/staged-recipes/pull/9996
- https://github.com/conda-forge/staged-recipes/pull/9997

Since `museval` package depends on `musdb` i need `musdb` package to be deployed first.